src/Data/List/Kleene/Base.agda

```diff
@@ -268,7 +266,7 @@ module _ (f : A → Maybe B → B) where
   foldrMaybe* []     = nothing
   foldrMaybe* (∹ xs) = just (foldrMaybe+ xs)
 
-  foldrMaybe+ xs = f (head xs) (foldrMaybe* (tail xs))
+  foldrMaybe+ (x & xs) = f x (foldrMaybe* xs)
```

Comment:
Weird that it used to be otherwise, but perhaps the termination checker has improved for mutual definitions since the original code was written?